addons/account/models/account_move.py

```diff
@@ -5806,6 +5806,7 @@ def action_print_pdf(self):
 
     def preview_invoice(self):
         self.ensure_one()
+        self.company_id.terms_type = "html"
```

Comment:
I think if that is always called here so it is fine to make that here in preview invoice to do not have the style that make the spaces between the words.

Reply:
Overwriting a company field/setting without any check or feedback to the user seems like a bad idea, especially just to solve a words' spacing issue (which could/should probably be done in CSS?).